airflow/cli/commands/standalone_command.py

```diff
@@ -193,9 +193,7 @@ def initialize_database(self):
             self.print_output("standalone", "Creating admin user")
             role = appbuilder.sm.find_role("Admin")
             assert role is not None
-            password = "".join(
-                random.choice("abcdefghkmnpqrstuvwxyzABCDEFGHKMNPQRSTUVWXYZ23456789") for i in range(16)
-            )
+            password = "".join(random.choices("abcdefghkmnpqrstuvwxyzABCDEFGHKMNPQRSTUVWXYZ23456789", k=16))
```

Comment:
Can we just replace this with `string.ascii_letters + string.digits` while we're in here?

Reply:
I think the string does not contain characters like `o0ijl1L` because they look bad in passwords one has to read and remember.

Reply:
Oooh good point! I was wondering why there were missing characters

Reply:
A comment might be a good idea here so this discussion doesn’t need to be repeated in the future.